eva/executor/load_executor.py

```diff
@@ -36,26 +34,24 @@ def validate(self):
 
     def exec(self):
         """
-        Read the input video using opencv and persist data
-        using storage engine
+        Use TYPE to determine the type of data to load. 
         """
+        
+        print(f"AbstractLoadDataExecutor: inside exec")
+
+        # TODO: Currently using file extension to decide which executor to invoke. Figure out how to get TYPE from the query
```

Comment:
I was not able to figure out how exactly a query is getting converted to a node plan, so I was not able to incorporate the "TYPE" keyword :/ I will need some more guidance here. Temporarily using file extension and invoking the corresponding executors.